hack/run-functests.sh

```diff
@@ -25,9 +27,20 @@ echo "Focusing on $FOCUS"
 
 export SUITES_PATH=cnf-tests/bin
 
-if ! cnf-tests/test-run.sh -ginkgo.focus=$FOCUS -junit /tmp/artifacts/ -report /tmp/artifacts/; then
-  failed=true
-  failures+=( "Tier 2 tests for $FEATURES" )
+if [ $CNF_TESTS_IMAGE != "" ]; then
+  cp -f $KUBECONFIG _cache/kubeconfig
+  echo "Running dockerized version via $CNF_TESTS_IMAGE"
+
+  if ! $IMAGE_RUN_CMD run -v $(pwd)/_cache/:/kubeconfig -v /tmp/artifacts/:/reports -e KUBECONFIG=/kubeconfig/kubeconfig $CNF_TESTS_IMAGE /usr/bin/test-run.sh \
```

Comment:
need to add :Z to kubeconfig
ie
```
-v $(pwd)/_cache/:/kubeconfig:Z
```
otherwise you get permission denied from within the container

Reply:
for reports too